src/core/converters/type_to_yaml.py

```
@@ -566,9 +593,18 @@ def _extract_dataclass_field_info(cls: type[Any]) -> dict[str, dict[str, Any]]:
 
     fields_info: dict[str, dict[str, Any]] = {}
 
+    # get_type_hints を使用して、文字列アノテーションと ForwardRef を解決
+    try:
+        resolved_hints = get_type_hints(cls)
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)